scripts/generate_can.py

```diff
@@ -192,6 +210,13 @@ def camel_to_snake_case(value):
                                 == 0 else 1/message['frequency']),
                 )
             )
+            # change value in "id:" section to hex
+            message['id'] = CAN_ID(message['id'])
+    
+    # converts CANID filter mask, CAN ID filters, and Roboteq CAN IDs to hex.
+    can_yaml[bus_name]['canid_filter_mask'] = CAN_ID(can_yaml[bus_name]['canid_filter_mask'])
```

Comment:
Does calling `int_keys_to_hex('canid_filter_mask')` not work here?

Reply:
With the way I've currently written the function, it won't since int_keys_to_hex only converts keys to hex and not the values (in the context of a python dictionary), for values I can directly set it to hex using the CAN_ID representer thing. Changing the keys and not the values are kinda wonky since you need to delete the old one and insert a new one, rather than just iterating through the values and converting each one.

Reply:
Alright, I'll create the PR in the other repo in a bit. :)